src/topology/algebra/ordered.lean

```diff
@@ -261,6 +283,48 @@ from (tendsto_infi.2 $ assume l, tendsto_infi.2 $ assume hl,
 
 end partial_order
 
+section decidable_linear_ordered_comm_group
+variable [decidable_linear_ordered_comm_group α]
+local attribute [instance] partial_order.topology
+local attribute [instance] partial_order.order_topology
+
+lemma decidable_linear_ordered_comm_group.order_topology.tendsto_iff_epsilon {β : Type*}
```

Comment:
I'm not sure about this name. Maybe https://github.com/leanprover-community/mathlib/blob/fd5617c11aeddbfe0fafdedd9e90d1de1f4a9dcc/src/topology/metric_space/basic.lean#L361-L391 suggests to stop at `tendsto_iff`

Reply:
I changed it to `tendsto_nhds` to be consistent with the theorems you referred to.